terragrunt/modules/docs-rs/web-server.tf

```diff
@@ -1,88 +1,92 @@
+locals {
+  web_domain = "web.${var.domain}"
+}
+
 resource "aws_security_group" "web" {
-  vpc_id = var.vpc_id
+  vpc_id = var.cluster_config.vpc_id
   name   = "docs-rs-web"
 }
 
-# TODO: configure web app
-# module "web" {
-#   source         = "../shared/modules/ecs-app"
-#   cluster_config = data.terraform_remote_state.shared.outputs.ecs_cluster_config
+module "web" {
+  source         = "../ecs-app"
+  cluster_config = var.cluster_config
 
-#   env  = "prod"
-#   name = "docs-rs-web"
-#   repo = "rust-lang/docs.rs"
+  env  = "dev"
```

Comment:
Why are we not using `staging` here to keep naming consistent with the account?

Reply:
This might sound more passive-aggressive than it was intended. 😅 I suspect there might be a historic reason why `dev` is used here. But if there isn't, I would strongly advise us to keep the naming consistent between environment variables and AWS accounts.

Reply:
That's a good point. `staging` is a much longer word, but perhaps that's fine?